pkg/server/ca/manager.go

```diff
@@ -74,6 +77,9 @@ type Manager struct {
 	nextJWTKey    *jwtKeySlot
 
 	journal *Journal
+
+	// Used to log a warning only once.
+	warnOnce sync.Once
```

Comment:
can you put a better name here? so we can be sure we are talking about warning when upstream authority does not support jwt?